app/src/main/res/layout/activity_customer_login.xml

```diff
@@ -0,0 +1,132 @@
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintHorizontal_bias="0.503"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/logo_customer_login_activity" />
+
+    <EditText
+        android:id="@+id/customer_login_activity_password_input_field"
+        android:layout_width="284dp"
+        android:layout_height="60dp"
+        android:background="@drawable/login_info_entry_box_background"
+        android:ems="10"
+        android:fontFamily="@font/mundial_light"
+        android:gravity="start|center_vertical"
+        android:hint="@string/password_entry_box_text"
+        android:inputType="textPassword"
+        android:paddingStart="16dp"
+        android:textColorHint="@color/dark_gray"
+        android:textSize="20sp"
+        app:layout_constraintBottom_toTopOf="@+id/customer_login_activity_no_account_question_text"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintHorizontal_bias="0.503"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/logo_customer_login_activity" />
+
+    <TextView
+        android:id="@+id/customer_login_activity_invalid_credentials_error"
+        android:layout_width="0dp"
+        android:layout_height="36dp"
+        android:ems="10"
+        android:fontFamily="@font/mundial_regular"
+        android:gravity="center"
+        android:inputType="text"
+        android:text="@string/customer_login_activity_invalid_credentials_error_text"
+        android:textColor="@color/red"
+        android:textSize="16sp"
+        app:layout_constraintBottom_toTopOf="@+id/customer_login_activity_no_account_question_text"
+        app:layout_constraintEnd_toEndOf="@+id/login_button_customer_login_activity"
+        app:layout_constraintHorizontal_bias="0.0"
+        app:layout_constraintStart_toStartOf="@+id/login_button_customer_login_activity"
+        app:layout_constraintTop_toBottomOf="@+id/login_button_customer_login_activity" />
+
+    <TextView
+        android:id="@+id/customer_login_activity_no_account_question_text"
+        android:layout_width="170dp"
+        android:layout_height="36dp"
+        android:ems="10"
+        android:fontFamily="@font/mundial_regular"
+        android:gravity="center"
+        android:inputType="text"
+        android:text="@string/no_account_question_text"
+        android:textColor="@color/dark_gray"
+        android:textSize="16sp"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/logo_customer_login_activity"
+        app:layout_constraintVertical_bias="0.76" />
+
+    <Button
+        android:id="@+id/customer_login_activity_create_account_button"
+        android:layout_width="150dp"
```

Comment:
"Create Account" text clips on default virtual device: https://i.snipboard.io/ze8ZtQ.jpg. Increasing width to 200dp seems to fix it.